.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2.1
+
+orbs:
+  win: circleci/windows@1.0.0
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      - build_test_windows:
+        filters:
+          branches:
+            only:
+              - master
+              - staging
+
+jobs:
+  build_test_windows:
+    executor: win/vs2019
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
```

Comment:
Is the fallback necessary?

Reply:
Not sure, I took this from the example config.